app/views/users/_sidebar_additional.html.erb

```diff
@@ -22,8 +22,6 @@
         </div>
       </a>
     <% end %>
-    <% if @user.badge_achievements_count > 0 %>
-      <%= render "articles/badges_area" %>
-    <% end %>
+    <%= render "articles/badges_area" %>
```

Comment:
^ This check was actually duplicated just inside the `badges_area` template ([here](https://github.com/thepracticaldev/dev.to/pull/3729/files#diff-ae027a3a438869c94fbb7f5ce2fcf576R1)), so I picked one and remove the other